app/models.py

```diff
@@ -1467,6 +1467,7 @@ def serialize(self):
         data.update({
             'id': self.id,
             'briefId': self.brief_id,
+            'briefTitle': self.brief.data.get('title', ''),
```

Comment:
Yeah having proper serializer classes would be a good move but it would be best to introduce as a global pattern and have a bit of discussion around it. I'll put something on the tech debt board to see if it's something the team would like to do going ahead.